### PR TITLE
Update some PyTorch examples for new API

### PR DESCRIPTION
**Pull request by PenghuiCheng**
Signed-off-by: Cheng, Penghui <penghui.cheng@intel.com>

## Type of Change

Update PyTroch examples(3d_unet, CNN-2, MobileNet_v2)
No API changed

## Description

JIRA ticket: https://jira.devtools.intel.com/secure/RapidBoard.jspa?rapidView=32982&projectKey=ILITV&view=detail&selectedIssue=ILITV-2354

### CODE REVIEW

**Comment by chensuyue** [comment]
1. example readme need to update.
2. benchmark part need update into Config Class as well.
3. run_benchmark.sh input parameter update.
4. json param missing main_script.

Details pls check INC newAPI examples.pptx

**Comment by chensuyue** [comment]
Pls rebase the PR.

File: examples/pytorch/nlp/blendcnn/distillation/eager/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This document describes the step-by-step instructions for reproducing PyTorch Bl
 ## 1. Installation
 
 ```Shell
-cd examples/pytorch/nlp/blendcnn/distillation/eager
+cd examples/pytorch/nlp/blendcnn/distillation/fx
```
Comment (chensuyue) [inline examples/pytorch/nlp/blendcnn/distillation/eager/README.md:11]: Change eager to fx? then, the path need to update too.

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3755/artifact/report.html)  
Please fix issues.

File: examples/pytorch/nlp/blendcnn/distillation/eager/README.md (hunk 1 above)
Comment (PenghuiCheng) [reply examples/pytorch/nlp/blendcnn/distillation/eager/README.md:11]: no need change

File: examples/.config/model_params_pytorch.json [hunk 2]
```diff
@@ -382,10 +367,7 @@
       "model_src_dir": "nlp/blendcnn/quantization/ptq/eager",
       "dataset_location": "/tf_dataset/pytorch/blendcnn/MRPC/",
       "input_model": "/tf_dataset/pytorch/blendcnn/models/",
-      "yaml": "blendcnn.yaml",
-      "strategy": "basic",
       "batch_size": 32,
-      "new_benchmark": false
```
[the remark this answers is not in the document]
Comment (PenghuiCheng) [reply examples/.config/model_params_pytorch.json:388]: It is old API, no need to add main_script.

File: examples/pytorch/nlp/blendcnn/distillation/eager/README.md (hunk 1 above)
Comment (PenghuiCheng) [reply examples/pytorch/nlp/blendcnn/distillation/eager/README.md:11]: no need to change

**Comment by PenghuiCheng** [comment]
extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3789/

**Comment by chensuyue** [comment]
> extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3789/

retest with expend tuning time for [3dunet,bert_large_SQuAD](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3802/)

**Comment by chensuyue** [comment]
Please check accuracy gap for gpt_j_wikitext. 
image

**Comment by chensuyue** [comment]
> > extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3789/
> 
> retest with expend tuning time for [3dunet,bert_large_SQuAD](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3802/)

image
bert_large_SQuAD evaluate too slow.

**Comment by PenghuiCheng** [comment]
extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3817/

**Comment by chensuyue** [comment]
> extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3817/

retest the model failed with connect issue: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3824/

**Comment by chensuyue** [comment]
> > extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3817/
> 
> retest the model failed with connect issue: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3824/

PLease check 3dunet issue.

**Comment by chensuyue** [comment]
> https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3824/

image 
What's wrong with the distilbert_base_ipex tuning result?

**Comment by PenghuiCheng** [comment]
extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3879/

**Review by chensuyue — APPROVED** [review]

**Comment by chensuyue** [comment]
> Please check accuracy gap for gpt_j_wikitext. image

Have you check this issue, it's still in the latest test. 
image

**Pull request merged**